djangoevents/tests/test_domain.py

```diff
@@ -53,3 +59,30 @@ def test_create_for_event():
 
     assert obj.id == 'ENTITY_ID'
     assert obj.version == 0
+
+
+@override_settings(BASE_DIR='/path/to/proj/src/')
+def test_version_1():
+    assert get_event_version(SampleEntity.Created) == 1
+
+
+def test_version_4():
+    try:
+        # make temporary directory structure
+        temp_dir = tempfile.mkdtemp()
+        entity_dir = os.path.join(temp_dir, 'sample_entity')
+        os.mkdir(entity_dir)
+
+        for version in range(1, 4):
+            # make empty schema file
+            expected_schema_path = os.path.join(entity_dir, 'v{}_sample_entity_created.json'.format(version))
+            with open(expected_schema_path, 'w'):
+                pass
+
+            # refresh version
+            set_event_version(SampleEntity, SampleEntity.Created, avro_dir=temp_dir)
+
+            assert get_event_version(SampleEntity.Created) == version
+    finally:
+        # remove temporary directory
+        shutil.rmtree(temp_dir)
```

Comment:
basically it's fine but I'd strongly advice to use [pytest's `tmpdir` fixture](https://docs.pytest.org/en/latest/tmpdir.html) - it would remove need to write this teardown and would simplify files creation

Reply:
Thanks, will do. 👍